src/core/etl/tests/Flow/ETL/Tests/Benchmark/RowsBench.php

```diff
@@ -31,4 +31,12 @@ public function bench_chunk_10() : void
 
         }
     }
+
+    #[Revs(5)]
+    public function bench_drop_100() : void
```

Comment:
hmm the name is a bit misleading, I would call it bench_drop_100_on_10k

Reply:
Renamed both in this class.